targets/lenovo-x1/appvms/element.nix

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+{pkgs, ...}: let
+  dendrite-pinecone = pkgs.callPackage ../../../packages/dendrite-pinecone {};
+in {
+  name = "element";
+
+  packages = [dendrite-pinecone pkgs.tcpdump pkgs.pamixer];
+  macAddress = "02:00:00:03:08:01";
+  ramMb = 4096;
+  cores = 4;
+  extraModules = [
+    {
+      # Enable pulseaudio for user ghaf to access mic
+      sound.enable = true;
+      hardware.pulseaudio.enable = true;
+      users.extraUsers.ghaf.extraGroups = ["audio"];
+
+      systemd.network = {
+        enable = true;
+        networks."10-ethint0" = {
+          DHCP = pkgs.lib.mkForce "no";
+          matchConfig.Name = "ethint0";
+          addresses = [
+            {
+              addressConfig.Address = "192.168.100.253/24";
+            }
+          ];
+          routes = [{routeConfig.Gateway = "192.168.100.1";}];
+          linkConfig.RequiredForOnline = "routable";
+          linkConfig.ActivationPolicy = "always-up";
+        };
+      };
+
+      networking = {
+        firewall.allowedTCPPorts = [dendrite-pinecone.TcpPortInt];
+        firewall.allowedUDPPorts = [dendrite-pinecone.McastUdpPortInt];
+      };
+
+      time.timeZone = "Asia/Dubai";
```

Comment:
Please set the timeZone for microvm via config like this https://github.com/tiiuae/ghaf/blob/eefaa2dd2865a36760d5ca07453bde7d1e2496b8/modules/desktop/graphics/waybar.config.nix#L55